autode/wrappers/MOPAC.py

```diff
@@ -198,10 +198,32 @@ def get_input_filename(self, calc):
     def get_output_filename(self, calc):
         return f'{calc.name}.out'
 
+    def get_version(self, calc):
+        """Get the version of MOPAC used to execute this calculation"""
+
+        for line in calc.output.file_lines:
+
+            if '(Version:' in line and len(line.split()) >= 3:
+                # e.g.        MOPAC2016 (Version: 19.144L)
+
+                try:
+                    name = line.split()[0]
+                    # Item between the brackets with only the version number
+                    version = line.split('(')[1].split(')')[0].split()[1]
+
+                    return f'{name} {version}'
+
+                except IndexError:
+                    pass
+
+        logger.warning('Could not get the version number from the output file')
+        return '???'
+
```

Comment:
is there no doi for MOPAC?

Reply:
not for MOAPAC2016, but I guess the original ref. should be there (https://link.springer.com/article/10.1007/BF00128336)